### ISSUE TITLE
Lambda config support

### ISSUE THREAD

**Issue by dougmoscrop** [aws/aws-xray-daemon#14]
It's not clear to me how this daemon works within Lambda, but would adding something like `/var/task/.xray/cfg.yaml` [here](https://github.com/aws/aws-xray-daemon/blob/master/daemon/cfg/cfg.go#L31) make it possible to configure e.g. sending X-Ray traces from Lambda to a different account?

**Comment by yogiraj07** [comment]
Hi @dougmoscrop ,
Lambda is responsible for starting and configuring the X-Ray daemon. Currently, Lambda uses the account that owns the function to post X-Ray data to, and does not have a way of configuring to post to an alternative account today.
We have this on our radar, and hope to work with the Lambda team to support this in the near future.

Hope this helps,
Yogi

**Comment by dougmoscrop** [comment]
Right, I intended this as a way to help users override the config without having Lambda have to support it

**Comment by dougmoscrop** [comment]
It's been another year; a lot of us are waiting for this support.  I can push through our enterprise TAMs if it helps, but it seems to me like if someone from Lambda and someone from X-Ray stat together for a couple hours they could agree to like, maybe let us write a Daemon configuration via a Lambda layer and just defer starting the daemon until after the layers have been loaded but before the runtime is bootstrapped.
